spec/gridstack-spec.js

```diff
@@ -244,6 +244,38 @@ describe('gridstack', function() {
         });
     });
 
+    describe('grid.cellHeight', function() {
+        beforeEach(function() {
+            document.body.insertAdjacentHTML(
+                'afterbegin', gridstackHTML);
+        });
+        afterEach(function() {
+            document.body.removeChild(document.getElementsByClassName('grid-stack')[0]);
+        });
+        it('should have no changes', function() {
+            var options = {
+                cellHeight: 80,
+                verticalMargin: 10,
+                width: 12
+            };
+            $('.grid-stack').gridstack(options);
+            var grid = $('.grid-stack').data('gridstack');
+            grid.cellHeight( grid.cellHeight() );
+            expect(grid.cellHeight()).toBe(80);
+        });
+        it('should change cellHeight to 120', function() {
+            var options = {
+                cellHeight: 80,
+                verticalMargin: 10,
+                width: 10
+            };
+            $('.grid-stack').gridstack(options);
+            var grid = $('.grid-stack').data('gridstack');
+            grid.cellHeight( 120 );
+            expect(grid.cellHeight()).toBe(120);
+        });
+    });
+
```

Comment:
added simple test above to call cellHeight() after the fact to get more coverage. I don't know how to test to make sure grid changes height when increasing row height (80->120) which would fail to work correctly in the nested case (but might be ok otherwise).